JS/HomeWork_4/extra/task3/script.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+let question = prompt("Enter the day of the week");
+if(question == 'Monday'){
+    for(let i = 1; i < 30; i+=7){
+        console.log(i)}
+}
+else if(question == 'Tuesday'){
+    for(let i = 2; i < 30; i+=7){
+        console.log(i)}
+}
+else if(question == 'Wednesday'){
+    for(let i = 3; i < 30; i+=7){
+        console.log(i)}
+}
+else if(question == 'Thursday'){
+    for(let i = 4; i < 30; i+=7){
+        console.log(i)}
+}
+else if(question == 'Friday'){
+    for(let i = 5; i < 30; i+=7){
+        console.log(i)}
+}
+else if(question == 'Saturday'){
+    for(let i = 6; i < 30; i+=7){
+        console.log(i)}
+}
+else if(question == 'Sunday'){
```

Comment:
If I wont type in lowercase?) And need some error message if we have wrong input